apps/main/src/loan/entities/mint-market-max-leverage.ts

```diff
@@ -15,8 +15,15 @@ type MarketParams = FieldsOf<MarketQuery>
 const _getMintMarketMaxLeverage = async ({ marketId }: MarketQuery): Promise<{ value: string }> => {
   const api = requireLib('llamaApi')
   const market = api.getMintMarket(marketId)
-  const maxLeverage = market.leverageV2.hasLeverage() ? await market.leverageV2.maxLeverage(market?.minBands) : ''
-  return { value: maxLeverage }
+  const hasMaxLeverage = market.leverageV2.hasLeverage()
+
+  if (hasMaxLeverage) {
+    const v2MaxLeverage = await market.leverageV2.maxLeverage(market?.minBands)
+    return { value: v2MaxLeverage }
+  }
+
+  const v1MaxLeverage = await market.leverage.maxLeverage(market?.minBands)
+  return { value: v1MaxLeverage }
```

Comment:
The `value` field is not necessary
```suggestion
  return hasMaxLeverage ? market.leverageV2.maxLeverage(minBands) : market.leverage.maxLeverage(minBands)
```